python/pecos/engines/cvm/binarray2.py

```diff
@@ -41,7 +43,8 @@ def set(self, value):
 
     def new_val(self, value):
         b = BinArray2(self.size, value, self.dtype)
-        b.clamp(self.size)
+        if self.dtype in (np.uint32, np.uint64):
```

Comment:
I wish I could include all unsigned types instead of enumerating them.

Reply:
@dlucchetti same question...